lib/compat/wordpress-6.1/persisted-preferences.php

```diff
@@ -23,7 +23,15 @@ function gutenberg_configure_persisted_preferences() {
 				'context' => array( 'edit' ),
 				'schema'  => array(
 					'type'                 => 'object',
-					'properties'           => array(),
+					'properties'           => array(
+						'__modified' => array(
+							'description' => __( 'The date and time the preferences were updated.', 'default' ),
+							'type'        => 'string',
+							'format'      => 'date-time',
+							'context'     => array( 'edit' ),
+							'readonly'    => true,
```

Comment:
Is `readonly` right here? We're modifying it via REST API put requests, no? Possible that I misunderstand what `readonly` means in this context 😀

Reply:
I managed to miss this comment and I've already clicked merge. I'll make a separate PR.

Reply:
All good—easier to iterate than lug this big PR around.

Let me know where you landed with https://github.com/WordPress/gutenberg/pull/39795#discussion_r854760887 too which is a comment that I forgot about 😀

Reply:
Github makes it too easy to miss these things.

You do get an uncaught promise error if the user is offline, or the REST API errors. It doesn't stop the editor from working or anything, so doesn't seem like a critical issue.

Local storage is there as a back up, so everything still works.

I'll work on a PR to see what the options are for handling this better.